packages/jupyter-ai/jupyter_ai/document_loaders/directory.py

```diff
@@ -37,6 +52,8 @@ def path_to_doc(path):
     ".jsx",
     ".tsx",
     ".txt",
+    ".html",
+    ".pdf",
```

Comment:
Currently, this has a mixture of uppercase and lowercase extensions. Should these be uniformly lowercase? (See other comments)

Reply:
Thanks, I made the necessary changes to the code. See
![Ensures all lower case comparison for PDF files](https://github.com/jupyterlab/jupyter-ai/assets/29005/ef673555-20d4-4751-8b84-bad45351c4e6)
![Lower case comparisons for all extensions](https://github.com/jupyterlab/jupyter-ai/assets/29005/9e3a5a34-dd30-46a0-aa4c-51f51e095dcb)
The test runs fine:
![Test for upper case file extensions](https://github.com/jupyterlab/jupyter-ai/assets/29005/f5ed26e0-af66-497f-b677-447d20279095)